dev-python/PythonQt/PythonQt-3.1.ebuild

```diff
@@ -0,0 +1,109 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_4 )
```

Comment:
dev-lang/python:3.5 is not stable for amd64, so I assume I should not support it until it become stable. I can test it under py3.5 and add support for 3.5 Should I?